### PR TITLE
Update driver latency optimization

### PR DESCRIPTION
**Pull request by Leo40Git**
Kudos to "-Wassy_-#7799".

### CODE REVIEW

File: docs/os/windows.md [hunk 1]
```diff
@@ -28,4 +28,7 @@ Use [TCP Optimizer](https://www.speedguide.net/downloads.php) to optimize your i
 2. Choose the optimal setting at the bottom.
 3. Click apply changes.
 
-Use [LatencyMon](https://www.resplendence.com/latencymon) and [DPC Latency Checker](https://www.thesycon.de/eng/latency_check.shtml) to analyze latency issues caused by kernel-mode device drivers. If they report issues, try updating your network drivers or installing alternate drivers.
+## Driver latency optimization
+
+Use [LatencyMon](https://www.resplendence.com/latencymon) and [DPC Latency Checker](https://www.thesycon.de/eng/latency_check.shtml) to analyze latency issues caused by kernel-mode device drivers. If they report issues, try updating your drivers or installing alternate drivers.
```
Comment (CrescentSpirit) [inline docs/os/windows.md:34]: Hello Leo!
Maybe you should add a note (after the latency issues part preferably) informing users how exactly those issues might affect them, e.g frequent stuttering and therefore, why such programs might be worth checking out; A small part to highlight their importance basically. 
Thanks!

Comment (Leo40Git) [reply docs/os/windows.md:34]: 98289a82ad2c149018f093b30e3c3aaa9d1b9d6c should resolve this.

File: docs/os/windows.md [hunk 2]
```diff
@@ -28,4 +28,8 @@ Use [TCP Optimizer](https://www.speedguide.net/downloads.php) to optimize your i
 2. Choose the optimal setting at the bottom.
 3. Click apply changes.
 
-Use [LatencyMon](https://www.resplendence.com/latencymon) and [DPC Latency Checker](https://www.thesycon.de/eng/latency_check.shtml) to analyze latency issues caused by kernel-mode device drivers. If they report issues, try updating your network drivers or installing alternate drivers.
+## Driver latency optimization
+
+Badly programmed kernel-mode device drivers might cause latency issues, which lead to problems such as frequent stuterring. The user must sniff out exactly which drivers are causing latency.
```
Comment (mastercoms) [inline docs/os/windows.md]: stuttering

**Review by mastercoms — CHANGES_REQUESTED** [review]
Just lil' typo

File: docs/os/windows.md (hunk 2 above)
Comment (Leo40Git) [reply docs/os/windows.md]: oops

a330fd056d7b527a1346db69d0b2a4fcf476b549

**Review by mastercoms — APPROVED** [review]

**Comment by mastercoms** [comment]
Thank you for your contribution to mastercomfig!

**Pull request merged**